### PR TITLE
68 api reference slu status

### PR DESCRIPTION
**Pull request by JuriBog**

### CODE REVIEW

**Review by jacekroszkowiakdev — APPROVED** [review]
I see no issues - good work!

File: slu-status/src/slu-status/pipes/slu-status-validation.pipe.ts [hunk 1]
```diff
@@ -12,6 +12,8 @@ export class SluStatusValidationPipe implements PipeTransform<SluStatusDto> {
 		if (errors.length > 0) {
 			throw new BadRequestException(errors);
 		}
+		console.log(object)
```
Comment (dominic22) [inline slu-status/src/slu-status/pipes/slu-status-validation.pipe.ts:15]: please remove logs

Comment (dominic22) [inline slu-status/src/slu-status/pipes/slu-status-validation.pipe.ts:15]: please remove logs

**Review by dominic22 — APPROVED** [review]

**Comment by dominic22** [comment]
can be merged after removing the logs 👍

**Pull request merged**